### PR TITLE
Adds a roboticist job only traitor item nanomachines

### PR DESCRIPTION
**Pull request by jughu**
Aka its the old nanomachines from the roburger that turns you into a cyborg comes in a bottle of 30 units cost 10 tc and is a virus with a reasonable amount of time which can be cured with copper

but since people probaly also get grammar nazi on me just pls make the desc for me

:cl: Hyena
add: Adds nanomachines bottle for roboticist traitors
/:cl:

[why]: # (Please add a short description [on the next line] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding:) NANOMACHINES FITS ROBOTICISTEE

### CODE REVIEW

**Comment by GunHog** [comment]
You opened this too late :cry: 
Feature freeze :sob:

**Comment by jughu** [comment]
i freeze and put this out the fridge when its summer

**Comment by coiax** [comment]
@lzimann can we get a frozen tag please?

**Comment by jughu** [comment]
@haevacht missed you hun 😊

**Comment by jughu** [comment]
I am back old friend

**Comment by Bawhoppen** [comment]
why would they buy this when they can just get infinite of it from the experimentor?

File: code/modules/uplink/uplink_item.dm [hunk 1]
```diff
@@ -1239,6 +1239,13 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
 	cost = 2
 	restricted_roles = list("Librarian")
 	limited_stock = 1 // please don't spam deadchat
+	
+/datum/uplink_item/role_restricted/nanomachines
+	name = "Nanomachines bottle"
```
Comment (coiax) [inline code/modules/uplink/uplink_item.dm]: Make It All Title Case

File: code/modules/uplink/uplink_item.dm [hunk 2]
```diff
@@ -1239,6 +1239,13 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
...
+	desc = "
```
Comment (coiax) [inline code/modules/uplink/uplink_item.dm]: Come on son, think of a fun description.

**Comment by Armhulen** [comment]
they can't get it from the experimentor anymore i believe

File: code/modules/uplink/uplink_item.dm [hunk 3]
```diff
@@ -1239,6 +1239,13 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
...
+	item = /obj/item/weapon/reagent_containers/glass/bottle/nanites
```
Comment (coiax) [inline code/modules/uplink/uplink_item.dm]: Make it like the romerol kit, make it come in a box with a syringe and dropper.

**Comment by XDTM** [comment]
even if you can spawn it you can't collect it

**Comment by Armhulen** [comment]
Also: What do they taste like? :thinking:

File: code/modules/uplink/uplink_item.dm [hunk 4]
```diff
@@ -1239,6 +1239,13 @@ var/list/uplink_items = list() // Global list so we only initialize this once.
...
+	cost = 10
```
Comment (coiax) [inline code/modules/uplink/uplink_item.dm]: Seems a little low, but it is traitor exclusive. Will be balance by how long the transformation period is, and how easy the cure is.

Comment (XDTM) [inline code/modules/uplink/uplink_item.dm]: copper is straight from the dispenser, but the toxin is fairly quick to act and not too noticeable. I think it should be ~14, in line with other easy-murder tools.

**Comment by Bawhoppen** [comment]
@Armhulen You can get them from the experimentor, just checked. You might be thinking of the xeno infection.

**Comment by PKPenguin321** [comment]
You can infect somebody with the nanomachines, then extract the virus with a Pandemic machine, meaning infinite borgs

Bad idea in action

**Comment by Bawhoppen** [comment]
@PKPenguin321 which I just explained why you already can do

**Comment by coiax** [comment]
@Bawhoppen Just because it makes smoke containing nanomachines doesn't mean you can extract it in liquid form and put it in the chef's cooking.

**Comment by Armhulen** [comment]
holy shit reeeeally

**Comment by Armhulen** [comment]
@coiax baws just explained how you can do that

**Comment by XDTM** [comment]
you cannot replicate chem viruses with the pandemic, i tried

**Comment by PKPenguin321** [comment]
Must have been recently fixed, I distinctly remember tasing criminals as captain and injecting them with borging blood

**Comment by Bawhoppen** [comment]
@everyone you get the virus from the experimentor, then syringe yourself, then eat some copper and you're good to go

**Comment by XDTM** [comment]
hey can you also make it not blacklisted if you do this

**Comment by KorPhaeron** [comment]
PRs that violated the freeze have to wait a week longer to be PRd or else people will keep opening their feature PRs during freezes

**Comment by jughu** [comment]
K kor

**Pull request closed without merging**